packages/pkgpanda-api/extra/dcos-pkgpanda-api.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=Pkgpanda API: Package Management Service
+Description=Pkgpanda API: installs and manages DC/OS components
```

Comment:
DC/OS Component Package Manager (Pkgpanda): installs and manages DC/OS components